.githooks/components.sh

```diff
@@ -7,7 +7,7 @@ components=( \
 	["features"]="feature-configs" \
 	["ztp"]="ztp","go.mod","go.sum" \
 	["infra"]="Makefile",".githooks/","hack/","openshift-ci/" \
-	["vendor"]="vendor","cnf-tests","ztp" \
-	["cnf-tests"]="cnf-tests","go.mod","go.sum"
+	["vendor"]="vendor","cnf-tests","ztp", "go.mod","go.sum" \
+	["cnf-tests"]="cnf-tests"
```

Comment:
I think go.mod and go.sum
belongs in ztp, cnf-tests and vendor

unless we want to block adding anything not through vendoring (good idea actually!)

and then please also remove it from ztp

Reply:
done